.github/ISSUE_TEMPLATE/check-in-experience-development-frontend.md

```diff
@@ -39,3 +39,10 @@ As a <user>, I want to <do something> so that I can <accomplish a goal>
 - [ ] Prod validation:
   - [ ] not necessary
   - [ ] scheduled
+
+## Validation Steps
```

Comment:
Could we maybe re-word this to be Code or Technical validation, to distinguish from our UX validation process?